go.mod

```diff
@@ -11,12 +11,14 @@ require (
 	github.com/edaniels/digest v0.0.0-20170923160545-b81e9c4ee11c
 	github.com/edaniels/golinters v0.0.3 // indirect
 	github.com/fatih/color v1.10.0
+	github.com/golang/mock v1.6.0 // indirect
```

Comment:
I'm going to guess the `// indirect` isn't actually true here and will use this moment to point out: it's been a minute since we ran `go mod tidy` in this codebase.  probably worth doing so now and cleaning things up a bit